package.json

```diff
@@ -21,7 +21,7 @@
     "charm": "^1.0.2",
     "google-auth-library": "^0.10.0",
     "googleapis": "^16.0.0",
-    "lighthouse": "paulirish/lighthouse#c74ea959",
+    "lighthouse": "denar90/lighthouse#14cd4f5",
```

Comment:
you can do  

    "lighthouse": "canary",

or 

    "lighthouse": "2.0.0-alpha.6",